scripts/start_local.sh

```diff
@@ -11,12 +11,12 @@ if [ "$1" = "y" ] || [ "$1" = "yes" ]; then
   ./scripts/build.sh
 fi
 
-./build/src/bedrock/monitoring &
+./build/src/bedrock/monitor/flmon &
```

Comment:
I assume this is more of my lack of C++ experience, but it seems like you wouldn't want your `build` directory to have the same paths as your `src` dirs? i.e., wouldn't you want `build/execs/flmon,flroute...`?

Reply:
By default cmake just places binaries in place in the build directory.  If we like we can specify a RUNTIME_OUTPUT_DIRECTORY. https://cmake.org/cmake/help/latest/command/add_executable.html

We should eventually have an INSTALL directive for make install as well. These seem orthogonal to this PR.